docs/release/release-process.md

```diff
@@ -192,6 +201,77 @@ success, all release artifacts will be officially published.
 After the release build succeeds, verify that the correctly versioned Stack images have been pushed
 to Docker Hub.
 
+### Template Stack Release Process
+
+The Template Stacks we maintain are slightly different from the
+controller-based stacks that we maintain. Their processes are similar
+but a little simpler. This section will walk through how to release the
+Template Stacks themselves, and does not directly apply to core
+Crossplane.
+
+For Template Stacks, we do not use release branches unless we need to
+make a patch release. In the future we may need a more robust branching
+strategy, but for now we are not using branches because it is simpler.
+
+Note that Template Stacks **do not** require any code changes to update
+the version.
+
+### Template Stack Tag And Publish Pipeline
+
+Here is the list of all template stacks:
+
+* `stack-minimal-gcp`
+* `stack-minimal-aws`
+* `stack-minimal-azure`
+* `app-wordpress`
+
```

Comment:
Could you add `templating-controller` release, too?

Reply:
I considered this, and I decided not to because we've been doing a pretty good job of keeping the releases decoupled. I think it could make sense to mention templating-controller here for completeness, but I'm thinking it makes sense for us to continue releasing the templating-controller separately. It makes it easier for us to manage the dependency if we're able to release it separately so that we don't need to update any versions when we release crossplane and the stacks.

Reply:
I've added some notes discussing the templating-controller project

Reply:
Yeah, makes sense.